modules/microshift-cni.adoc

```diff
@@ -0,0 +1,301 @@
+
+== Getting a snapshot of OVS interfaces from a running cluster
+.Procedure
+To see a snapshot of OVS interfaces from a running {product-title} cluster, use the following command:
+
+[source, terminal]
+----
+$ sudo ovs-vsctl show
+----
+
+.Example OVS interfaces in a running cluster
+[source, terminal]
+----
+9d9f5ea2-9d9d-4e34-bbd2-dbac154fdc93
+    Bridge br-ex
+        Port enp1s0
+            Interface enp1s0
+                type: system
+        Port br-ex
+            Interface br-ex
+                type: internal
+        Port patch-br-ex_localhost.localdomain-to-br-int <1>
+            Interface patch-br-ex_localhost.localdomain-to-br-int
+                type: patch
+                options: {peer=patch-br-int-to-br-ex_localhost.localdomain} <1>
+    Bridge br-int
+        fail_mode: secure
+        datapath_type: system
+        Port patch-br-int-to-br-ex_localhost.localdomain
+            Interface patch-br-int-to-br-ex_localhost.localdomain
+                type: patch
+                options: {peer=patch-br-ex_localhost.localdomain-to-br-int}
+        Port eebee1ce5568761
+            Interface eebee1ce5568761 <2>
+        Port b47b1995ada84f4
+            Interface b47b1995ada84f4 <2>
+        Port "3031f43d67c167f"
+            Interface "3031f43d67c167f" <2>
+        Port br-int
+            Interface br-int
+                type: internal
+        Port ovn-k8s-mp0 <3>
+            Interface ovn-k8s-mp0
+                type: internal
+    ovs_version: "2.17.3"
+----
+<1> The `patch-br-ex_localhost.localdomain-to-br-int` and `patch-br-int-to-br-ex_localhost.localdomain` are OVS patch ports that connect `br-ex` and `br-int`.
+<2> The pod interfaces `eebee1ce5568761`, `b47b1995ada84f4` and `3031f43d67c167f` are are named with the first 15 bits of pod sandbox ID and are plugged in the `br-int` bridge.
+<3> The OVS internal port for hairpin traffic,`ovn-k8s-mp0` is created by the `ovnkube-master` container.
+
+== Primary gateway interface
+The `microshift-ovs-init.service` uses your specified host interface for the cluster network. You can specify the desired host interface name in a hint file `/var/lib/ovnk/iface_default_hint`. The specified interface will be added in OVS bridge br-ex which acts as gateway bridge for the CNI network.
+//Q: is this the primary gateway interface?
+//procedure?
+
+== Secondary gateway interface
+With the `microshift-ovs-init.service`, you can set up one additional host interface for cluster ingress and egress. This is done by adding the additional host interface name in another hint file, `/etc/ovnk/extra_bridge`.
+//steps?
```

Comment:
Again the hint file isn't used anymore and we have `externalGatewayInterface:`in the ovn yaml